appservice/src/createAppService/AppServicePlanSkuStep.ts

```diff
@@ -12,9 +12,11 @@ import { AppKind, WebsiteOS } from './AppKind';
 import { IAppServiceWizardContext } from './IAppServiceWizardContext';
 import { setLocationsTask } from './setLocationsTask';
 
+type ExtendedSkuDescription = WebSiteManagementModels.SkuDescription & { label?: string; description?: string }
```

Comment:
Why make the label/descriptions optional though?  I feel like we should enforce them if you're making an ExtendedSkuDescription.

Reply:
Well I'm using the same type for both the basic and advanced scenarios. In this case "Extended" means, "hey you can add better labels if you want to", which only applies for basic create